packages/main/src/themes/SuggestionItem.css

```diff
@@ -0,0 +1,14 @@
+:host([ui5-suggestion-item]) {
+	height: auto;
+	min-height: var(--_ui5_list_item_base_height);
+}
+
+:host([ui5-suggestion-item]) .ui5-li-root {
+	min-height: var(--_ui5_list_item_base_height);
+}
+
+:host([ui5-suggestion-item]) .ui5-li-content {
+	padding-bottom: .875rem;
```

Comment:
don't we have a parameter about this?

Reply:
It behaves the same as in the main branch. I noticed that there is no difference between compact and cozy styles which looks strange to me. IMHO this is an existing bug and we should fix it with another change following latest specs of the list.